### DISCUSSION TITLE
`Images in this message are hidden. This message might be suspicious or spam.` in gmail when sending transactional email with resend/react email

### DISCUSSION THREAD

**Discussion by dlinardi** [resend/react-email#1807] [Q&A]
[![enter image description here](https://i.sstatic.net/zOzcvc05.png)](https://i.sstatic.net/zOzcvc05.png)

I'm experiencing an issue with Gmail hiding images in transactional emails sent through Resend, despite having proper DNS configuration and verification.

**Setup:**
- Turborepo monorepo deployed on Vercel
- Multiple apps in `/apps` folder, each with its own subdomain (e.g., `https://a.example.com`, `https://b.example.com`)
- Using [Resend](https://resend.com/) for transactional emails with [React Email](https://react.email) templates
- All DNS records properly configured (verified green checks in Resend dashboard - Insights tab)
- Domain and email authentication are properly set up

**Issue:**
When Gmail users receive the transactional emails, images are hidden by default with the message:
> "Images in this message are hidden. This message might be suspicious or spam."

**Important Details:**
- Emails are delivered to inbox (not spam)
- Only one image per email (company logo)
- No tracking pixels or suspicious content
- Image is stored in the app's `/public` folder
- Image URL format: `https://b.example.com/logo.png`
- Other email clients that I've tested display images correctly
- Resend dashboard shows all green checks for email delivery and authentication
- Only Gmail exhibits this behaviour
- Clicking `Show Images` does work and render the company logo as expected

**Example Email Template:**
```jsx
<Img
  src={`${getBaseUrl()}/logo.png`}
  width="146"
  height="40"
  alt="Company Logo"
  title="Company Logo"
/>
```

I tried:
1. Following Resend's documentation for proper DNS setup and domain authentication (SPF, DKIM, DMARC)
2. Verifying the image URL matches the sending domain
3. Setting proper image dimensions and alt/title tags
4. Ensuring image is served over HTTPS
5. Checking Resend's dashboard insights which show all green checks for email deliverability

Expected behavior:
- Images should display automatically in Gmail, just like they do in other email clients (Outlook, Apple Mail)
- No "Images are hidden" warning message

Actual behavior:
- Gmail hides images by default with warning: "Images in this message are hidden. This message might be suspicious or spam."
- Users must click "Show images" to see the company logo
- Other email clients display images correctly without any warnings

**Comment by gabrielmfern** [comment]
You might be better off to talk with Resend's support on this, if it does end up being an issue with React Email we all communicate internally so they will let me know. I'll share this on the internal Slack to see if anyone jumps in to help you, but it seems like an issue unrelated to React Email for now.

**Comment by Gomah** [comment]
We were experiencing the same issue, and somehow disabling `Click Tracking` fixed it for us
